### PR TITLE
Improve styling for feature page

### PR DESCRIPTION
**Pull request by jmezach**
Various small improvements to the feature page so that it becomes a more rounded feature set.

### CODE REVIEW

File: src/Augurk.UI/Shared/MainLayout.razor [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
     <div class="main">
         <div class="top-row px-4">
-            <a href="https://augurk.github.io" target="_blank" class="ml-md-auto">About</a>
+
```
Comment (marktaling) [inline src/Augurk.UI/Shared/MainLayout.razor:15]: Do we still want to place this one somewhere else?

File: src/Augurk.UI/wwwroot/index.html [hunk 2]
```diff
@@ -33,6 +33,23 @@
     <script src="lib/underscore.js/underscore-min.min.js"></script>
     <script src="lib/webfont/webfontloader.js"></script>
     <script src="lib/js-sequence-diagrams/dist/sequence-diagram-min.min.js"></script>
+
+    <script lang="javascript">
+        $(function(){
```
Comment (marktaling) [inline src/Augurk.UI/wwwroot/index.html:38]: Perhaps we should add some comments here? Or is this fine?

**Review by marktaling — APPROVED** [review]

File: src/Augurk.UI/Shared/MainLayout.razor (hunk 1 above)
Comment (jmezach) [reply src/Augurk.UI/Shared/MainLayout.razor:15]: Maybe we should have a footer as well where we can place this link?

File: src/Augurk.UI/wwwroot/index.html (hunk 2 above)
Comment (jmezach) [reply src/Augurk.UI/wwwroot/index.html:38]: Probably ;).

File: src/Augurk.UI/Shared/MainLayout.razor (hunk 1 above)
Comment (jmezach) [reply src/Augurk.UI/Shared/MainLayout.razor:15]: Let's leave this for now and we'll revisit in a new PR.

**Pull request merged**